app/internal/page/certificateprovider/your_address.go

```diff
@@ -0,0 +1,85 @@
+package certificateprovider
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/page"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/page/form"
```

Comment:
What do you think about putting this at `github.com/ministryofjustice/opg-modernising-lpa/internal/form` instead? I was kind of thinking of keeping `page/*` as the different journeys. ...and later down the line we might move `validation` into or under `form`.

Reply:
That makes sense - I'll rejig out to its own internal package